src/components/LinkButtons/LinkButtons.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import styles from './LinkButtons.module.scss';
+import ArrowRight from '../../img/Arrow.inline.svg';
+
+const LinkButtons = ({buttons}) => {
+	return(
+		<div className={styles.wrap}>
+			{
+				buttons.map((item, i)=>{
+					if (i%2===0) {
+						return(
+							<button className="button is-primary">
```

Comment:
Why create a new LinkButtons-component? How is this different from a regular Button? https://github.com/SEALAB-Software/international-website/blob/master/src/components/Button/Button.js

Reply:
This is create for http://prntscr.com/svua4d.
We can add buttons in the section as possible as we can.

Reply:
But I still think you could use the existing Button-component?

Reply:
Fixed in the commit 4090e6f